code/modules/reagents/reagents/toxin.dm

```diff
@@ -437,18 +437,30 @@
 	meltprob = 30
 	taste_multi = 1.5
 
+/datum/reagent/toxin/nanites
+	name = "Nanomachines"
+	description = "Microscopic construction robots designed to tear iron out of the surroundings and build jagged structures of wire when mixed into a foam. Drinking this is a bad idea."
+	taste_description = "poor life choices, followed by burning agony"
+	reagent_state = LIQUID
+	color = "#535E66" // rgb: 83, 94, 102
+
+/datum/reagent/toxin/nanites/on_mob_life(mob/living/L, metabolism)
+	L.apply_damages(10*REM, 6*REM, 6*REM) //DO NOT DRINK THIS. Seriously!
+	L.blood_volume -= 10
+	return ..()
+
 /datum/reagent/toxin/xeno_neurotoxin
 	name = "Neurotoxin"
 	description = "A debilitating nerve toxin. Impedes motor control in high doses. Causes progressive loss of mobility over time."
 	reagent_state = LIQUID
 	color = "#CF3600" // rgb: 207, 54, 0
-	custom_metabolism = REAGENTS_METABOLISM * 2 //otherwise it defaults to half of that
+	custom_metabolism = REAGENTS_METABOLISM * 2
 	purge_list = list(/datum/reagent/medicine)
 	purge_rate = 1
 	overdose_threshold = REAGENTS_OVERDOSE
 	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL * 1.2 //make this a little more forgiving in light of the lethality
 	scannable = TRUE
-	toxpwr = 0
+	toxpwr = 0.8
```

Comment:
uhhh why neuro is literally supposed to be nonlethal

Reply:
this is also not documented in CL

Reply:
It IS documented in the CL (AND in the commit).

>Xeno neurotoxin now does minor toxin. 0.4 damage/tick, 1 sting is 10 damage. Might be more acceptable now, we'll see.

As to "why", I noticed that neurotoxin doesn't seem to do much currently, so I gave it minimal permanent effects.

Reply:
Its supposed to be nonlethalfor a reason per design this is a bad idea so remove it

Reply:
Ok, I'll make a PR to change it later when I have to fix chemxeno.

Reply:
done